### PR TITLE
Update RHV CA certificate command

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1868742

CP for 4.5, 4.6

### CODE REVIEW

File: modules/virt-importing-vm-wizard.adoc [hunk 1]
```diff
@@ -41,11 +41,13 @@ ifdef::virt-importing-rhv-vm[]
 ** *API URL*: For example, `\https://<RHV_Manager_FQDN>/ovirt-engine/api`
 ** *CA certificate*: Click *Browse* to upload the RHV Manager CA certificate or paste the CA certificate into the field.
 +
-You can obtain the CA certificate by running the following command:
+View the CA certificate by running the following command:
 +
 ----
```
Comment (aburdenthehand) [inline modules/virt-importing-vm-wizard.adoc:46]: Not strictly related to the PR but while we're here, can you please add the [source,terminal] metadata to this codeblock, as per:
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output
Thank you!

Comment (apinnick) [reply modules/virt-importing-vm-wizard.adoc:47]: done

**Pull request merged**